frontend/payment-klarna/components/Checkout.vue

```diff
@@ -23,17 +23,17 @@ export default {
     LoadingSpinner
   },
   async mounted () {
-    await this.upsertOrder()
-    // Todo: refactor
     this.$bus.$on('kcoOrderLoaded', () => {
       setTimeout(async () => {
         const order = await this.$store.dispatch('kco/fetchOrder', this.checkout.orderId)
         this.onKcoAddressChange({
           totalSegments: this.totals.total_segments,
           shippingAddress: order.shipping_address
         })
-      }, 2000)
+      }, 500)
     })
+
+    await this.upsertOrder()
```

Comment:
Any reason for this being moved? This will happen before the `setTimeout` either way.